packages/react-reconciler/src/ReactFiberNewContext.js

```diff
@@ -691,10 +691,22 @@ export function readContext<T>(context: ReactContext<T>): T {
   return readContextForConsumer(currentlyRenderingFiber, context);
 }
 
+/**
+ * @deprecated This is function is deprecated and will be removed in a future release.
+ * Please use `readContextDuringReconciliation(...)` instead.
+ */
```

Comment:
Though this fn seems to be private, I deprecated it just in case it's public.
Lmk if it's private or not and okay to directly rename it and I'll revert this deprecation diff.

Reply:
ping @acdlite 
since you added this fn can you comment on the above?